### PR TITLE
Add PulumiUP demo videos to pages

### PR DESCRIPTION
**Pull request by gthuang**
## Description
Adding demo videos of Review Stacks, Property Search, TF conversion, and Azure Native to blogs, docs, and webpages.

## Checklist:

- [x] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [x] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

**Review by mnlumi — CHANGES_REQUESTED** [review]
Adding these to the top of the blog posts is not a great look on the preview page ([Ref: Avoid using images or code samples in the first 70 words](https://github.com/pulumi/pulumi-hugo/blob/master/themes/default/archetypes/blog-post/index.md?plain=1#L50-L52)). Also, it seems kind of odd to drop a video as the first line of the blog post without any context or lead-in.

Screenshot 2023-06-29 at 8 53 00 AM
Screenshot 2023-06-29 at 8 53 19 AM
Screenshot 2023-06-29 at 8 53 08 AM

**Comment by gthuang** [comment]
> Adding these to the top of the blog posts is not a great look on the preview page ([Ref: Avoid using images or code samples in the first 70 words](https://github.com/pulumi/pulumi-hugo/blob/master/themes/default/archetypes/blog-post/index.md?plain=1#L50-L52)). Also, it seems kind of odd to drop a video as the first line of the blog post without any context or lead-in.
> 
> Screenshot 2023-06-29 at 8 53 00 AM Screenshot 2023-06-29 at 8 53 19 AM Screenshot 2023-06-29 at 8 53 08 AM

i made these changes

File: themes/default/content/docs/pulumi-cloud/deployments/review-stacks.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ menu:
     weight: 2
 ---
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/VvQcx51YL4g" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
```
Comment (mnlumi) [inline themes/default/content/docs/pulumi-cloud/deployments/review-stacks.md:14]: This needs to be moved.

File: themes/default/content/blog/review-stacks/index.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ tags:
     - review stacks
 ---
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/VvQcx51YL4g" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
```
Comment (mnlumi) [inline themes/default/content/blog/review-stacks/index.md:18]: This needs to be moved.

**Review by mnlumi — COMMENTED** [review]
The videos need to be moved out of the beginning of all blog posts.

**Review by mnlumi — APPROVED** [review]

**Pull request merged**